definitions/pipelines/immuno.cwl

```diff
@@ -12,6 +12,11 @@ requirements:
     - class: SubworkflowFeatureRequirement
 inputs:
     #rnaseq inputs
+    reference_rnaseq:
```

Comment:
Is it necessary for this to be a separate reference than the existing `reference` input?

Reply:
I dont see any reference (.fa) passed to the RNA seq pipeline. So to make sure the RNA seq pipelines has it i had to add it to the immuno pipeline as well.

Reply:
hrmm - you're right.  The "reference_index" param right below that is a pointer to the hisat indexes (like `/gscmnt/gc2560/core/GRC-human-build38_human_95_38_U2AF1_fix/rna_seq_annotation/hisat2.1.0_index/GRCh38DH`, which don't necessarily have the fasta adjacent to them.  If we swap this out for the star pipeline in the future, it'll get passed the `star_genome_dir` (which contains the fasta IIRC) and we could drop this param, but it does seem necessary here, I think.

Reply:
To clarify, I am not asking about whether a reference needs to be passed to rnaseq.  I'm wondering if this parameter could be used:
https://github.com/genome/analysis-workflows/blob/b465f0da2806ddb6df481409541d13288ccb40ec/definitions/pipelines/immuno.cwl#L57-L61
rather than creating a new `reference_rnaseq` input.  The existing `reference` input is used for several of the other subworkflows in `immuno.cwl`.

Reply:
Right that makes sense. Fixed it.